openslides/users/views.py

```diff
@@ -68,6 +71,7 @@ def update(self, request, *args, **kwargs):
             # Check permissions only to update yourself.
             if str(request.user.pk) != self.kwargs['pk']:
                 self.permission_denied(request)
+            #del request.data['groups_id']
```

Comment:
Now the user can update himself/herself. But in the request data are things like `groups_id`, `is_committee`, `username`, `is_present`... A user without `users.can_manage` is not allowed to change these entries. The form forbids to change them, but you can manually create a request and change you groups etc.

In the previous lines is checked witch permissions the user has. If he has not the right permissions, the special data should be removed. When commenting in `del request.data['groups_id']` I get an error, that `groups_id` is required. Is there a possibility just to partially update the user or strip out the forbidden fields?